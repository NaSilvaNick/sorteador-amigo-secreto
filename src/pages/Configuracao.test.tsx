import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil"
import Configuracao from './Configuracao'

const mockNavegacao = jest.fn();

jest.mock('react-router-dom', () => ({
  useNavigate: () => mockNavegacao
}))

describe('', () => {
  test('Deve ser renderizado corretamente', () => {
    const { container } = render(
      <RecoilRoot>
        <Configuracao />
      </RecoilRoot>
    )

    expect(container).toMatchSnapshot()
  })
})