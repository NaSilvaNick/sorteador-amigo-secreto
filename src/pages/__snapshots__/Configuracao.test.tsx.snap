// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Deve ser renderizado corretamente 1`] = `
<div>
  <form>
    <input
      placeholder="Insira os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
    
  </form>
  <ul />
  <footer>
    <button
      disabled=""
    >
      Iniciar brincadeira
    </button>
  </footer>
</div>
`;
